### ISSUE TITLE
Use node_env PROD for build

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/metamask-extension#4464]
Error says:

```
ui.js:formatted:90233 You are currently using minified code outside of NODE_ENV === 'production'. This means that you are running a slower development build of Redux. You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify or DefinePlugin for webpack (http://stackoverflow.com/questions/30030031) to ensure you have the correct code for your production build.
n.default @ ui.js:formatted:90233

```

**Comment by kumavis** [comment]
this is blocked by 
 - removing browserify transforms from package.json which is blocked by
   - removing test:flat
   - removing or rewriting mascara as an e2e test

**Comment by kumavis** [comment]
added blockers to zenhub

**Comment by whymarrh** [comment]
Fixed in #4983
